web/src/App.vue

```diff
@@ -0,0 +1,143 @@
+<!--
+Copyright 2022 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<template>
+  <v-app id="app">
+    <nav class="navbar">
+      <v-app-bar app flat>
+        <div class="turbinia-logo">
+          <v-img src="./assets/turbinia-logo-mark.png" max-height="50" max-width="70" contain />
+        </div>
+        <div class="turbinia-title">
+          <v-toolbar-title>Turbinia</v-toolbar-title>
+        </div>
+        <v-spacer></v-spacer>
+        <div class="dark-theme-btn">
+          <v-tooltip left>
+            <template v-slot:activator="{ on, attrs }">
+              <v-btn icon v-on:click="toggleTheme" v-bind="attrs" v-on="on">
+                <v-icon>mdi-brightness-6</v-icon>
+              </v-btn>
+            </template>
+            Switch to dark mode
+          </v-tooltip>
+        </div>
+      </v-app-bar>
+    </nav>
+    <v-main>
+      <div>
+        <v-card>
+          <v-card-title>
+            Request List
+            <v-spacer></v-spacer>
+            <v-text-field
+              v-model="search"
+              append-icon="mdi-magnify"
+              label="Search"
+              single-line
+              hide-details
+            ></v-text-field>
+            <v-spacer></v-spacer>
+            <v-tooltip left>
+              <template v-slot:activator="{ on, attrs }">
+                <v-btn fab color="blue lighten-2" @click="getRequest()" v-bind="attrs" v-on="on">
+                  <v-icon>mdi-refresh</v-icon>
+                </v-btn>
+              </template>
+              Refresh Request List
+            </v-tooltip>
+          </v-card-title>
+          <v-data-table
```

Comment:
Yep agreed. I was planning to, but left it out for this PR. Since functionality is all there now though, went ahead and moved it into it's own component under `components\RequestList.vue`